packages/vaadin-text-field/vaadin-text-field.d.ts

```diff
@@ -1,2 +1 @@
 export * from './src/vaadin-text-field.js';
-export * from './src/interfaces';
```

Comment:
Not sure what happened with this file, would be great if you can double check that this is not needed anymore @web-padawan

Reply:
My bad, forgot to remove it. Now when all the fields are in separate packages this file is not needed.